<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>to_do</class>
 <widget class="QWidget" name="to_do">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>To Do List</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Icons/icons.qrc">
    <normaloff>:/Logo/toDo.png</normaloff>:/Logo/toDo.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>1</number>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QPushButton" name="pb_delete_all">
     <property name="text">
      <string>Delete All</string>
     </property>
     <property name="icon">
      <iconset resource="../Icons/icons.qrc">
       <normaloff>:/Buttons/remove.png</normaloff>:/Buttons/remove.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QListWidget" name="list_widget"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pb_delete_item">
     <property name="text">
      <string>Delete Item</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pb_add_item">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Add Item</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
